templates/etc/icinga2/conf.d/downtimes.conf.j2

```diff
@@ -6,19 +6,17 @@
  */
 
 {% for downtime in icinga2_master_downtimes %}
-apply ScheduledDowntime "downtime-{{ downtime.name }}" to Service {
+apply ScheduledDowntime "downtime-{{ downtime.name }}" to {{ 'Service' if downtime.services else 'Host' }} {
   comment = "{{ downtime.comment }}"
```

Comment:
Maybe we should add the "author" as well.

Reply:
Yes, indeed - the `author` attribute is actually mandatory; Icinga2 will throw a syntax error if it's missing for an `apply` declaration.

Will fix this in #97.